### PR TITLE
Add Minecraft

### PR DESCRIPTION
**Pull request by g-plane**
**https://github.com/bs-community/awesome-minecraft**

**Minecraft is a famous sandbox video game and it has a wonderful eco-system like mods and other softwares.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -344,6 +344,7 @@
 - [Game Boy Development](https://github.com/avivace/awesome-gbdev#readme)
 - [Construct 2](https://github.com/armaldio/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
+- [Minecraft](https://github.com/bs-community/awesome-minecraft) - Sandbox video game.
```
Comment (styfle) [inline readme.md]: This link should should be https://github.com/bs-community/awesome-minecraft#readme

Comment (styfle) [reply readme.md]: The list seems a bit inconsistent.

- Some of the descriptions end with a period and some do not
- Some of the descriptions are written in Chinese and some are not
	- wiki.vg is missing a description
	- For example, one description says "(Chinese)" written in English

Comment (styfle) [reply readme.md]: Also missing a couple things:

- No project logo
- No code of conduct

Comment (g-plane) [reply readme.md]: Is logo necessary?

**Comment by sirredbeard** [comment]
From the outset looks like it meets the requirements.

File: readme.md (hunk 1 above)
Comment (styfle) [reply readme.md]: The logo is no necessary but it is strongly suggested.

**Comment by styfle** [comment]
The list seems a bit inconsistent.

- Some of the descriptions end with a period and some do not
- Some of the descriptions are written in Chinese and some are not
  - wiki.vg is missing a description
  - For example, one description says "(Chinese)" written in English

Also missing a couple things:

- No project logo
- No code of conduct

**Comment by g-plane** [comment]
@styfle Yes, code of conduct should be added and I will add it later. As for the project logo, that isn't necessary. Also, I will add all missing end periods. Last, the requirements doesn't limit the languages, so using English and Chinese both for descriptions is okey because some websites are in Chinese and they don't have English descriptions.

**Comment by kypiching** [comment]
I think this pull request can be close.

**Comment by g-plane** [comment]
They're all in English now.

**Comment by g-plane** [comment]
I don't have the permission to use the logo of Minecraft which owned by Mojang. For those who want the logo of awesome-minecraft, you can send a PR to [bs-community/awesome-minecraft](https://github.com/bs-community/awesome-minecraft).

**Comment by sindresorhus** [comment]
The list is lacking content. The Minecraft community is huge. There's a lot more awesome content than what's in this list.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

Not done.

**Comment by sindresorhus** [comment]
Don't include empty sections. Either find content for it or remove it until you have any content.

**Comment by sindresorhus** [comment]
> I don't have the permission to use the logo of Minecraft which owned by Mojang.

Use some artwork or screenshot from the game then.

**Comment by g-plane** [comment]
> The list is lacking content. The Minecraft community is huge. There's a lot more awesome content than what's in this list.

Yes but it needs some time to collect more content.

---
I will try to meet the left requirements.

**Comment by g-plane** [comment]
I have updated and please tell me again if there is something missing.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/bs-community/awesome-minecraft
✖ Linting

  readme.md:15:3
  ✖  15:3  ToC missing item for "Tweaking"  remark-lint:awesome/toc

  1 error
```

**Comment by sindresorhus** [comment]
> Includes a project logo/illustration whenever possible.
> **The image should be high-DPI. Set it to maximum half the width of the original image.**

**Comment by g-plane** [comment]
@sindresorhus Both are fixed.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1100471420650278917

**Pull request merged**